libs/api/Verifier.cpp

```diff
@@ -82,14 +82,15 @@ int
 load_byte_code(
     const char* filename,
     const char* sectionname,
+    bool use_mock_fd,
```

Comment:
I think it would be more future proof to pass in ebpf_verifier_options_t*

Reply:
done.